go.sum

```diff
@@ -1,5 +1,8 @@
-github.com/SAP/go-dblib v0.0.0-20210129100507-ad8fcb5232aa h1:7sR1xksKUl0n3k1Mtm1yOdGc5B+NmqQUxtcMXR5cHjY=
-github.com/SAP/go-dblib v0.0.0-20210129100507-ad8fcb5232aa/go.mod h1:+J5fa7eSdQ3ig2CqOarENCW3L2vPdDF+hl0N5h5pXOU=
+# SPDX-FileCopyrightText: 2021 SAP SE
+#
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
This header breaks go-lint. I'd rather accept a temporarily breaking license/reuse check instead of a permanent fail of go-lint.

Reply:
Resolved (see https://github.com/SAP/go-ase/pull/195)